packages/terra-time-input/src/_AccessibleInput.jsx

```diff
@@ -6,6 +6,10 @@ import { v4 as uuidv4 } from 'uuid';
 import TimeUtil from './TimeUtil';
 
 const propTypes = {
+  /**
+   * String that labels the current element. 'aria-label' must be present for accessibility.
+   */
+  ariaLabel: PropTypes.string,
```

Comment:
why is this added? TimeInput is currently not passing this

Reply:
Yes thanks, have made the change https://github.com/cerner/terra-framework/pull/2031/commits/dbacc7e8f6ee54df11ab377b0bf50949bf093e3e